scripts/update_lambda_in_dev.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Run in an authenticated shell for pay-dev.
+npm run build
+
+aws lambda update-function-code \
+  --function-name dev-log-stream-transformation \
+  --zip-file fileb://./dist/pay-logging-firehose-transformation-v0.0.1.zip
```

Comment:
This won't work if we start increasing the version number of the lambda. Can we use * in place of version?

Reply:
Yeah ok good thinking, I'll see if that works.